src/Pact/Gas/Table.hs

```diff
@@ -232,21 +238,26 @@ tableGasModel gasConfig =
         GUse _moduleName _mHash -> (_gasCostConfig_useModuleCost gasConfig)
           -- The above seems somewhat suspect (perhaps cost should scale with the module?)
         GInterfaceDecl _interfaceName _iCode -> (_gasCostConfig_interfaceCost gasConfig)
-        GUserApp t -> case t of
-          Defpact -> (_gasCostConfig_defPactCost gasConfig) * _gasCostConfig_functionApplicationCost gasConfig
-          _ -> _gasCostConfig_functionApplicationCost gasConfig
   in GasModel
       { gasModelName = "table"
       , gasModelDesc = "table-based cost model"
       , runGasModel = run
       }
+{-# INLINE tableGasModel #-}
 
 
-perByteFactor :: Decimal
-perByteFactor = 0.1
+perByteFactor :: Rational
```

Comment:
One of the goals here was to ensure `Rational` doesn't break compat so I don't think we want #760 merged ... but if you document here that this change doesn't break compat (ie, point to a commit with the golden _before_ the merge of 760), we can go with the merge.